pyiceberg/catalog/hive.py

```diff
@@ -150,6 +150,7 @@ def _construct_hive_storage_descriptor(schema: Schema, location: Optional[str])
 PROP_TABLE_TYPE = "table_type"
 PROP_METADATA_LOCATION = "metadata_location"
 PROP_PREVIOUS_METADATA_LOCATION = "previous_metadata_location"
+DEFAULT_PROPERTIES = {'write.parquet.compression-codec': 'zstd'}
```

Comment:
We only want to set this when writing Parquet, but that's the only format that we support today :)